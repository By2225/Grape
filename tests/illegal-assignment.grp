fun Int main() {
    Int a;
    Node<Int> b; 

    a = 3;
    b = 'a';

    a = b;
}
