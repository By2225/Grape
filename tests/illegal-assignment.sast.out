Fatal error: exception Failure("illegal assignment Int = Node<Int> in a = b")
